specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/storage.json

```diff
@@ -2184,6 +2184,34 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$maxpagesize",
+            "in": "query",
+            "type": "string",
```

Comment:
Should the $maxpagesize be a number? 
And is there a limitation for the number?

Reply:
changed it to integer type